src/pages/team/index.jsx

```diff
@@ -1,13 +1,23 @@
 import React from 'react';
 
 
-const Index = props => {
+import DisplayTeams from './sections/coremembers/index';
+import { team } from '../../data/airtable'
+import OurStory from './sections/ourstory/index';
+
+
 
-    return (
-        <div>
 
-        </div>
+const Team = () => {
+
+    const data = team.useTeam()
+    console.log(data)
+    return (
+        <>
+            <OurStory/>
+       
+            <DisplayTeams members={data} />
```

Comment:
As you are sending data inside the `displayteams` component create `PropTypes` for it.

Reply:
yes will create after next PR

Reply:
This is just file structure fixes